src/ncp/ncp_uart.cpp

```diff
@@ -136,6 +137,8 @@ void NcpUart::EncodeAndSendToUart(Tasklet &aTasklet)
 void NcpUart::EncodeAndSendToUart(void)
 {
     uint16_t len;
+    bool sendImmediate = false;
```

Comment:
I think we don't need the local `sendImmediate`...

Reply:
Yes, will fix.